### ISSUE TITLE
cli panics when running hello-world example

### ISSUE THREAD

**Issue by anubhavmishra** [dapr/cli#163]
## Expected Behavior

```bash
dapr run --app-id mynode --app-port 3000 --port 3500 node app.js --kubernetes
```

```bash
Starting Dapr with id mynode on port 3500
```

## Actual Behavior

```bash
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1ce0154]

goroutine 1 [running]:
github.com/dapr/cli/cmd.glob..func4(0x2b98e40, 0xc000168d80, 0x2, 0x9)
        /Users/vsts/agent/2.158.0/work/1/s/gopath/src/github.com/dapr/cli/cmd/run.go:65 +0x104
github.com/spf13/cobra.(*Command).execute(0x2b98e40, 0xc000168cf0, 0x9, 0x9, 0x2b98e40, 0xc000168cf0)
        /Users/vsts/agent/2.158.0/work/1/s/gopath/src/github.com/dapr/cli/vendor/github.com/spf13/cobra/command.go:766 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0x2b984c0, 0x1cdf884, 0x2b984c0, 0xc00041ff60)
        /Users/vsts/agent/2.158.0/work/1/s/gopath/src/github.com/dapr/cli/vendor/github.com/spf13/cobra/command.go:852 +0x2c0
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/vsts/agent/2.158.0/work/1/s/gopath/src/github.com/dapr/cli/vendor/github.com/spf13/cobra/command.go:800
github.com/dapr/cli/cmd.Execute(0x20ef610, 0x5, 0x20ef568, 0x3)
        /Users/vsts/agent/2.158.0/work/1/s/gopath/src/github.com/dapr/cli/cmd/dapr.go:39 +0x88
main.main()
        /Users/vsts/agent/2.158.0/work/1/s/gopath/src/github.com/dapr/cli/main.go:19 +0x51
```

**Comment by yaron2** [comment]
Can you try again and remove the --kubernetes flag?

**Comment by anubhavmishra** [comment]
yea, that works!

**Comment by anubhavmishra** [comment]
I think the `output` is `nil` in https://github.com/dapr/cli/blob/master/cmd/run.go#L51

**Comment by yaron2** [comment]
Yeah you're basically not supposed to use --kubernetes with the run command.. `dapr run` launches dapr and your app side by side locally.

**Comment by anubhavmishra** [comment]
Ah! Interesting, How can I run that example on my Kubernetes cluster? I have already done `dapr init --kubernetes`. Is there a way from the CLi to deploy the example app into Kubernetes.

**Comment by yaron2** [comment]
We were thinking about how to do that, but that would mean Dapr would need to know how to package your app into a container and deploy it..

Look at the following end to end sample that'll show you how to deploy to Kubernetes:

https://github.com/dapr/samples/tree/master/2.hello-kubernetes

**Comment by anubhavmishra** [comment]
oh I see the `hello-kubernetes` example.

https://github.com/dapr/samples/tree/master/2.hello-kubernetes

**Comment by LMWF** [comment]
@yaron2 should the --kubernetes option be removed from `dapr run` then?

**Comment by yaron2** [comment]
very much so

**Comment by LMWF** [comment]
Can this issue be closed?  I opened https://github.com/dapr/cli/issues/195 to remove --kubernetes from dapr run.

**Comment by LMWF** [comment]
It looks like this question is resolved, #195 was opened to address the separate comment about `dapr run --kubernetes`.
